src/components/routes/Host/ListingForm/ListingForm.container.ts

```diff
@@ -37,20 +37,23 @@ const ListingFormContainer = styled.section`
           > * {
             margin-bottom: 16px;
           }
-          .row-address {
-            display: flex;
-            flex-direction: row;
-            justify-content: space-between;
-            .bee-input-wrapper {
-              &:nth-of-type(1) {
-                width: 45%;
-              }
-              &:nth-of-type(2) {
+          >.bee-input-wrapper {
+            &:nth-of-type(3) {
```

Comment:
what are these nth type for?  is it for the ordering style?
if it's for targeting certain fields like `title`, they should be attached via class names since fields can change order.

Reply:
3, 4, and 5 are for the `city`, `state`, and `postalCode` styles respectively:

![image](https://user-images.githubusercontent.com/18321302/51953209-595e8280-23f0-11e9-830c-06783411ae96.png)

I could attach classNames instead but attributes like the `margin-left` would be constrained to that input so if a dev switched the order, the margins would be off. Originally had a div that wrapper the three with class `row-address` that took care of the styles/spacing in that row.

@kcvan suggested this to `map` through an array of objects to render the address inputs: https://github.com/thebeetoken/beenest-web/pull/95#discussion_r251677561

Reply:
can we add these attributes to a class that lives at the `bee-input-wrapper` level? that would make the `nth-of-type` css sensible. otherwise a programmer wouldn't know why 
```
>.bee-input-wrapper {
            &:nth-of-type(3) {
``` 
is related to an address formatting.

Reply:
added `.city`, `.state`, `.postal-code` within `.address`